sdk/oko_sdk_core/src/methods/open_modal.ts

```diff
@@ -110,10 +115,52 @@ function openEmailLoginPopup(this: OkoWalletInterface): PopupContext {
 
   const popupOrigin = new URL(this.sdkEndpoint).origin;
 
+  // Wait for nonce to be set, then redirect popup to actual email login URL
```

Comment:
This function I remember shouldn't use async method. Safari issue. Is nonce necessary?

Reply:
Popup opens immediately (`about:blank`) to bypass Safari blocker, then waits for nonce setup before redirect. Nonce is optional for Auth0 but added for security.